### DISCUSSION TITLE
Any Ideas on how to make the guide.xml publicly available?

### DISCUSSION THREAD

**Discussion by Mauricio300808** [iptv-org/community#572] [Q&A]
Hello, the guide works great for me but only when im on my local network.... so, It would be great to access my guide on my devices when I travel.. I thought about uploading the xml file periodically to GitHub but I'm not sure if that's the best way and don't even know where to start :) any ideas???

**Comment by Slee42069** [comment]
you can use npm run serve :D i run npm forever and run serve so the files are constantly available also with cron jobs updating the xml files each day

**Reply by Mauricio300808** [reply]
Thank you @Slee42069 im currently using npm run but I'd like to be able to use the guide outside of my local network..

**Reply by DylanKarimagoko** [reply]
Any luck bud im stuck on deployment im try to deploy to vercel but i cant find the entry point for the api index.ts or index.js

**Reply by Mauricio300808** [reply]
no luck...

**Reply by JReming85** [reply]
You can either use Cloudflare Zero Trust, to expose it externally, a reverse proxy thats exposed from your home router, or just use port forwarding from your router to the epg server

**Comment by PopeyeTheSai10r** [comment]
To make your guide publicly accessible, you will need to upload them to a site or cloud storage.
Some popular options are AWS S3 bucket and Azure blob storage.

**Comment by freearhey** [comment]
https://github.com/iptv-org/epg?tab=readme-ov-file#access-the-guide-by-url
